packages/packages.go

```diff
@@ -64,6 +66,7 @@ type Packages struct {
 	Deb           []PkgInfo     `json:"deb,omitempty"`
 	Zypper        []PkgInfo     `json:"zypper,omitempty"`
 	ZypperPatches []ZypperPatch `json:"zypperPatches,omitempty"`
+	Cos           []PkgInfo     `json:"cos,omitempty"`
```

Comment:
We aren't super diligent about this, but I assume you may prefer COS over Cos

Reply:
Fixed in the new commit.